File: Web.Admin/src/api/services/merchant/models/QueryMerchantDetailOutput.ts
import { PaymentChannelEnum } from "@/api/enums/PaymentChannelEnum";

/**
 * Fast.Center.Service.Merchant.Dto.QueryMerchantDetailOutput 获取商户号详情输出
 */
export interface QueryMerchantDetailOutput {
  /**
   * 商户号Id
   */
  merchantId?: number;
  /**
   * 
   */
  merchantType?: PaymentChannelEnum;
  /**
   * 商户号
   */
  merchantNo?: string;
  /**
   * 商户密钥
   */
  merchantSecret?: string;
  /**
   * 公钥序号
   */
  publicSerialNum?: string;
  /**
   * 公钥
   */
  publicKey?: string;
  /**
   * 证书序号
   */
  certSerialNum?: string;
  /**
   * 证书
   */
  cert?: string;
  /**
   * 证书私钥
   */
  certPrivateKey?: string;
  /**
   * 备注
   */
  remark?: string;
  /**
   * 创建者用户名称
   */
  createdUserName?: string;
  /**
   * 创建时间
   */
  createdTime?: Date;
  /**
   * 更新者用户名称
   */
  updatedUserName?: string;
  /**
   * 更新时间
   */
  updatedTime?: Date;
  /**
   * 更新版本控制字段
   */
  rowVersion?: number;
}

File: Api.Server/src/Services/Center/Merchant/Dto/QueryMerchantDetailOutput.cs
namespace Fast.Center.Service.Merchant.Dto;

/// <summary>
/// <see cref="QueryMerchantDetailOutput"/> 获取商户号详情输出
/// </summary>
public class QueryMerchantDetailOutput : PagedOutput
{
    /// <summary>
    /// 商户号Id
    /// </summary>
    public long MerchantId { get; set; }

    /// <summary>
    /// 商户号类型
    /// </summary>
    public PaymentChannelEnum MerchantType { get; set; }

    /// <summary>
    /// 商户号
    /// </summary>
    public string MerchantNo { get; set; }

    /// <summary>
    /// 商户密钥
    /// </summary>
    public string MerchantSecret { get; set; }

    /// <summary>
    /// 公钥序号
    /// </summary>
    public string PublicSerialNum { get; set; }

    /// <summary>
    /// 公钥
    /// </summary>
    public string PublicKey { get; set; }

    /// <summary>
    /// 证书序号
    /// </summary>
    public string CertSerialNum { get; set; }

    /// <summary>
    /// 证书
    /// </summary>
    public string Cert { get; set; }

    /// <summary>
    /// 证书私钥
    /// </summary>
    public string CertPrivateKey { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }
}